aries-site/src/examples/templates/dashboards/components/ActivityFeed.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Text,
+  ThemeContext,
+} from 'grommet';
+import { useContext } from 'react';
+
+export const ActivityFeed = ({ children, title, footer: footerContent }) => {
+  const theme = useContext(ThemeContext);
+  const { body, footer, header } = theme.card;
```

Comment:
Noticing we are changing the padding below -- do you think these changes could potentially be applicable across all cards? or it's specific to this use case?

Reply:
So far specific to this use case.